static/js/setup.js

```diff
@@ -39,10 +39,5 @@ $(function () {
             }
             return this;
         };
-
-        $.fn.within = function (sel) {
```

Comment:
Did you intend to remove this code?  If so, it's probably best to do it as a separate commit.